ServiceBus/ServiceBus/CampaignService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity.Migrations;
+using System.Linq;
+using SharedLibs.DataContracts;
+
+namespace ServiceBus
+{
+    public class CampaignService : ICampaignService
+    {
+        public SharedLibs.DataContracts.Campaign GetCampaign(Guid guid)
+        {
+            //before continuing is necessary to add Campaign to ServiceBusDatabaseEntities
+            throw new NotImplementedException();

```

Comment:
I would preffer having something like:

return new Result.Fatal("Not Implemented");

This should not throw an exception, but rather give the client information that the service method is not implemented yet. Please do that for all methods. I will approve after